src/App.tsx

```diff
@@ -20,12 +26,17 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={visibleMovies}
               />
             </div>
           </div>
         </div>
 
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={moviesFromServer.filter(movie => (
+          movie.description.toLowerCase().includes(query.trim().toLowerCase())
+          || movie.title.toLowerCase().includes(query.trim().toLowerCase())))}
+        />
```

Comment:
```suggestion
        <MoviesList movies={filteredMovies}
        />
```
please create a function for filtering movies, don't put that much logic inside JSX